contracts/contracts/coordination/Coordinator.sol

```diff
@@ -253,6 +274,7 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
         ritual.endTimestamp = ritual.initTimestamp + duration;
         ritual.accessController = accessController;
 
+        uint96 minAuthorization = application.minimumAuthorization();
```

Comment:
I'd suggest to move this variable either to immutable or to variable, each call is slower because of using inner call. I know that is not crucial but would be good to make

Reply:
✅